src/gtest/test_sidechain_certificate_quality.cpp

```diff
@@ -358,7 +397,115 @@ TEST_F(SidechainsMultipleCertsTestSuite, V2Cert_LowerQuality_SameEpoch_Sidechain
     EXPECT_TRUE(sidechain.balance                     == initialScState.balance);
 }
 
-TEST_F(SidechainsMultipleCertsTestSuite, V2Cert_LowerQuality_DifferentEpoch_SidechainIsUpdated) {
+TEST_F(SidechainsMultipleCertsTestSuite, V2Cert_DifferentEpoch_SameQuality_SidechainIsUpdated) {
```

Comment:
Yes, I agree with this. I would leave the zero quality submission test as it is ok in non-ceasing v2.

Reply:
Done in c7bb0cc7d304d7a51cc8ee874108797c8a3debe4

Reply:
Structured binding definitely improves readability
I suggest to enforce it by
```
const auto [bestCertEpoch, bestCertQuality] = mBestCertDataByScId[scid];
```
and then refer to variables rather than `first` and `second`

Reply:
I was not aware of the structured binding feature introduced in C++17, thanks; I will definitely looking into that.
However, in this context this looks like a task more related to refactoring, than to quality revert. For this reason, I prefer to postpone the enforcing of this practice and leave the PR as it is now.